ncm-metaconfig/src/main/metaconfig/haproxy/pan/schema.pan

```diff
@@ -21,6 +21,7 @@ type haproxy_service_global_config = {
     'daemon' : string = ''
     'pidfile' : string = '/var/run/haproxy.pid'
     'chroot' : string = '/var/lib/haproxy'
+    'log-send-hostname' ? string
```

Comment:
`type_hostname`?

Reply:
actually, what I'm trying to achieve is either:
```
log-send-hostname somestringyoucaninvent
```
and
```
log-send-hostname
```
the latter is directing haproxy to guess the hostname itself.

Reply:
Hmm, ok!